src/components/characterPreview/CharacterScoringSummary.tsx

```diff
@@ -52,45 +50,46 @@ export const CharacterScoringSummary = (props: { simScoringResult: SimulationSco
     )
   }
 
-  function ScoringNumber(props: { label: string; number: number; precision?: number }) {
-    const precision = props.precision ?? 1
-    const value = props.number ?? 0
-    const show = value != 0
-    return (
-      <Flex align='center' gap={15}>
-        <pre style={{ margin: 0 }}>{props.label}</pre>
-        <pre style={{ margin: 0 }}>{show && value.toFixed(precision)}</pre>
-      </Flex>
-    )
-  }
-
-  function ScoringInteger(props: { label: string; number: number }) {
-    const value = props.number ?? 0
-    return (
-      <Flex align='center' gap={1}>
-        <pre style={{ margin: 0 }}>{props.label}</pre>
-        <pre style={{ margin: 0 }}>{value}</pre>
-      </Flex>
-    )
-  }
-
   function ScoringAbility(props: { comboAbilities: string[]; index: number }) {
-    const displayValue = props.comboAbilities[props.index]
+    const displayValue = i18n.exists(`charactersTab:CharacterPreview.BuildAnalysis.Rotation.${props.comboAbilities[props.index]}`)
+      // @ts-ignore ts is being dumb :/, key existence is verified on the line above and for some reason can't tell that t() always returns a string
+      ? t(`CharacterPreview.BuildAnalysis.Rotation.${props.comboAbilities[props.index]}`)
+      : null
     if (displayValue == null) return <></>
 
     return (
       <Flex align='center' gap={15}>
-        <pre style={{ margin: 0 }}>{`#${props.index} - ${displayValue}`}</pre>
+        <pre style={{ margin: 0 }}>{`#${props.index} - ${displayValue as string}`}</pre>
       </Flex>
     )
   }
 
-  function ScoringText(props: { label: string; text: string }) {
-    const value = props.text ?? ''
+  function ScoringDisplayColumn(props: {
+    values: { label: string; value: number | string; precision?: number }[]
+    style?: React.CSSProperties
+    rowSpacing?: React.CSSProperties['gap']
+    columnSpacing?: React.CSSProperties['gap']
+    justify?: React.CSSProperties['justifyContent']
+    align?: React.CSSProperties['alignItems']
+  }) {
```

Comment:
Not a fan of passing css properties like this, I feel like if it needs to be customized like this it should just be split into other components: ScoringTextA / ScoringTextB